from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_signup(object):
    def setupUi(self, signup):
        signup.setObjectName("signup")
        signup.resize(844, 843)
        self.centralwidget = QtWidgets.QWidget(signup)
        self.centralwidget.setObjectName("centralwidget")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(-1, -1, 841, 811))
        self.frame.setStyleSheet("background-color: rgb(255, 0, 0);\n"
"border-radius:15px;")
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.frame_2 = QtWidgets.QFrame(self.frame)
        self.frame_2.setGeometry(QtCore.QRect(110, 90, 621, 641))
        self.frame_2.setStyleSheet("background-color: rgb(255, 255, 255);\n"
"border-radius:15px;")
        self.frame_2.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.lineEdit = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit.setGeometry(QtCore.QRect(80, 150, 451, 61))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit.setFont(font)
        self.lineEdit.setStyleSheet("background-color: #d8d8d8;\n"
"")
        self.lineEdit.setObjectName("lineEdit")
        self.lineEdit_2 = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit_2.setGeometry(QtCore.QRect(80, 230, 451, 61))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit_2.setFont(font)
        self.lineEdit_2.setStyleSheet("background-color: rgb(216, 216, 216);")
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.lineEdit_3 = QtWidgets.QLineEdit(self.frame_2)
        self.lineEdit_3.setGeometry(QtCore.QRect(80, 310, 451, 61))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(12)
        self.lineEdit_3.setFont(font)
        self.lineEdit_3.setStyleSheet("background-color:#d8d8d8;")
        self.lineEdit_3.setEchoMode(QtWidgets.QLineEdit.Password)
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.label = QtWidgets.QLabel(self.frame_2)
        self.label.setGeometry(QtCore.QRect(240, 0, 151, 91))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(25)
        self.label.setFont(font)
        self.label.setStyleSheet("color: rgb(0, 0, 0);")
        self.label.setObjectName("label")
        self.pushButton = QtWidgets.QPushButton(self.frame_2)
        self.pushButton.setGeometry(QtCore.QRect(170, 410, 281, 71))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(17)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton.setFont(font)
        self.pushButton.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton.setStyleSheet("background-color:rgb(255, 0, 0);")
        self.pushButton.setObjectName("pushButton")
        self.label_2 = QtWidgets.QLabel(self.frame_2)
        self.label_2.setGeometry(QtCore.QRect(90, 550, 321, 71))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(17)
        font.setBold(True)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.pushButton_2 = QtWidgets.QPushButton(self.frame_2)
        self.pushButton_2.setGeometry(QtCore.QRect(430, 560, 101, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(19)
        font.setUnderline(True)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_2.setObjectName("pushButton_2")
        self.close = QtWidgets.QPushButton(self.frame)
        self.close.setGeometry(QtCore.QRect(800, 10, 31, 28))
        self.close.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.close.setStyleSheet("background-color: rgb(193, 0, 0);\n"
"border-radius:8px;\n"
"")
        self.close.setText("")
        self.close.setObjectName("close")
        self.back = QtWidgets.QPushButton(self.frame)
        self.back.setGeometry(QtCore.QRect(10, 10, 41, 41))
        self.back.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.back.setStyleSheet("background-color:rgb(170, 85, 0);")
        self.back.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../Pictures/icons8-return-100.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.back.setIcon(icon)
        self.back.setIconSize(QtCore.QSize(27, 50))
        self.back.setObjectName("back")
        self.min = QtWidgets.QPushButton(self.frame)
        self.min.setGeometry(QtCore.QRect(720, 10, 31, 28))
        self.min.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.min.setStyleSheet("background-color: rgb(0, 229, 0);\n"
"border-radius:8px;")
        self.min.setText("")
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.min.setIcon(icon1)
        self.min.setObjectName("min")
        self.max = QtWidgets.QPushButton(self.frame)
        self.max.setGeometry(QtCore.QRect(760, 10, 31, 28))
        self.max.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.max.setStyleSheet("background-color: rgb(255, 255, 0);\n"
"border-radius:8px;")
        self.max.setText("")
        self.max.setIcon(icon1)
        self.max.setObjectName("max")
        signup.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(signup)
        self.statusbar.setObjectName("statusbar")
        signup.setStatusBar(self.statusbar)

        self.retranslateUi(signup)
        self.close.clicked.connect(signup.close)
        self.min.clicked.connect(signup.showMinimized)
        self.max.clicked.connect(signup.showNormal)
        QtCore.QMetaObject.connectSlotsByName(signup)

    def retranslateUi(self, signup):
        _translate = QtCore.QCoreApplication.translate
        signup.setWindowTitle(_translate("signup", "MainWindow"))
        self.lineEdit.setToolTip(_translate("signup", "fill out this field"))
        self.lineEdit.setPlaceholderText(_translate("signup", "Username"))
        self.lineEdit_2.setToolTip(_translate("signup", "fill out this field"))
        self.lineEdit_2.setPlaceholderText(_translate("signup", "E-mail"))
        self.lineEdit_3.setToolTip(_translate("signup", "fill out this field"))
        self.lineEdit_3.setPlaceholderText(_translate("signup", "Password"))
        self.label.setText(_translate("signup", "sign up"))
        self.pushButton.setText(_translate("signup", "create account"))
        self.pushButton.setShortcut(_translate("signup", "Return"))
        self.label_2.setText(_translate("signup", "have already an account?"))
        self.pushButton_2.setText(_translate("signup", "sign in"))
        self.close.setToolTip(_translate("signup", "close"))
        self.back.setToolTip(_translate("signup", "Click to go back"))
        self.min.setToolTip(_translate("signup", "minimize"))
        self.max.setToolTip(_translate("signup", "normal"))

